src/collections/_documentation/platforms/node/aws_lambda.md

```diff
@@ -7,40 +7,41 @@ Create a deployment package on your local machine and install the required depen
 
 Install our Node.js SDK using `npm`:
 
-```basic
-npm install @sentry/node
+```bash
+$ npm install @sentry/node
```

Comment:
Need to check what syntax highlighting lib we use, but normally `$` goes with ` ```console`.
Sometimes `$` is visually good, but has the disadvantage of making it slightly harder to copy-paste.

bash:

```bash
$ npm install @sentry/node
results
```

console:

```console
$ npm install @sentry/node
results
```

```shell
npm install @sentry/node
```

^^ no `$` allows you to double click to select the whole line, copy, paste.

Reply:
True, but I wanted to follow what we already use in majority of examples (31 matches for `$ npm install`, 10 matches for `npm install`)